Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Events/InteractableEvent.cs

```diff
@@ -114,10 +131,13 @@ public static EventLists GetEventTypes()
                 }
             }
 
-            EventLists lists = new EventLists();
-            lists.EventTypes = eventTypes;
-            lists.EventNames = names;
-            return lists;
+            eventLists = new EventLists();
```

Comment:
With eventlists being static, won't subsequent Interactables reset the previous controls events?

Reply:
This method just compiles a list of available ReceiverBase types. That only needs to be done once. I cache the results in a static field so subsequent requests return the cached types instead.

Reply:
The list is created ad-hoc while in the editor which allows for new scripts to be added to the list, but once the app starts running, caching it is the right call. New scripts cannot be added during runtime. This was an oversight on my part.

Reply:
The list gets purged when the project recompiles so editor behavior should remain the same.